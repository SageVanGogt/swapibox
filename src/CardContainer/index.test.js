import React from 'react';
import { shallow } from 'enzyme';

describe('Card Container component', () => {
  it('should', () => {
    
  })
})