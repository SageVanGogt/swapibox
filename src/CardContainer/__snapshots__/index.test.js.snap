// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Container component should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardContainer
    currentSectionData={
        Array [
            Object {
              "model": "rambo",
              "name": "rambo",
              "passengerCount": "rambo",
              "vehicleClass": "rambo",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Card
          model="rambo"
          name="rambo"
          passengerCount="rambo"
          vehicleClass="rambo"
/>,
      ],
      "className": "card-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "card0",
        "nodeType": "function",
        "props": Object {
          "model": "rambo",
          "name": "rambo",
          "passengerCount": "rambo",
          "vehicleClass": "rambo",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Card
            model="rambo"
            name="rambo"
            passengerCount="rambo"
            vehicleClass="rambo"
/>,
        ],
        "className": "card-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "card0",
          "nodeType": "function",
          "props": Object {
            "model": "rambo",
            "name": "rambo",
            "passengerCount": "rambo",
            "vehicleClass": "rambo",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
